Description: remove dependency to is-wsl until it is in Debian
 See: https://salsa.debian.org/rust-team/debcargo-conf/-/merge_requests/588
Author: Blair Noctis <n@sail.ng>
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,4 +42,4 @@

-[target."cfg(any(target_os = \"linux\", target_os = \"android\", target_os = \"freebsd\", target_os = \"dragonfly\", target_os = \"netbsd\", target_os = \"openbsd\", target_os = \"illumos\", target_os = \"solaris\" ))".dependencies.is-wsl]
-version = "0.4.0"
+#[target."cfg(any(target_os = \"linux\", target_os = \"android\", target_os = \"freebsd\", target_os = \"dragonfly\", target_os = \"netbsd\", target_os = \"openbsd\", target_os = \"illumos\", target_os = \"solaris\" ))".dependencies.is-wsl]
+#version = "0.4.0"

--- a/src/unix.rs
+++ b/src/unix.rs
@@ -11,6 +11,6 @@

-    let wsl_path = wsl_path(path);
-    if is_wsl::is_wsl() {
-        commands.push(("wslview", vec![&wsl_path]));
-    }
+    //let wsl_path = wsl_path(path);
+    //if is_wsl::is_wsl() {
+    //    commands.push(("wslview", vec![&wsl_path]));
+    //}
